integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -125,6 +130,21 @@ class CarbonEnv {
     LOGGER.info("Initialize CarbonEnv completed...")
   }
 
+
+  def createIndexServerTmp(sparkSession: SparkSession): Unit = {
+    val conf = sparkSession.sparkContext.hadoopConfiguration
+    val fs = FileSystem.get(conf)
+    if (fs.getUri.toString.startsWith("hdfs://")) {
```

Comment:
Use CarbonUtil.checkAndAppendFileSystemURIScheme to add URI to the path

Reply:
done